_posts/2016-08-20-letsencrypt.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Moving To Let's Encrypt TLS Certificate"
+description: ""
+category:
+tags: []
+---
+{% include JB/setup %}
+
+As a fan of the EFF, and security in general, I was pretty excited to here about
+the [Let's Encrypt](https://letsencrypt.org/) project.   Let's Encrypt is a project sponsored by EFF, University of Michigan, Mozilla,
+Cisco and Akamai to provide free, signed TLS certificates.  While I've used
+StartSSL in the past for free certificates, I've found their process a bit
+cumbersome (although in all fairness, they have done a ton of redesign this year).  Despite the redesign, I went forward with using Let's Encrypt to renew my TLS certificate.
+
+I used the [EFF Certbot](https://certbot.eff.org/) as my client, which works fairly well,
+even on an older CentOS system.  It couldn't quite parse the Apache
+configuration correctly, but pointing to the certificate manually was not
+a hard configuration change.  In case anyone finds the configuration handy,
+here it the appropriate lines from my httpd.conf below.
+
+Hopefully this begins a new exciting chapter in secure Internet communication
```

Comment:
my brain was expecting this to say "an exciting new chapter" and was sad when it was wrong